cmd/install/assets/recordingrules/hypershift.yaml

```diff
@@ -74,4 +74,10 @@ groups:
     expr: max by(platform) (hypershift_hostedclusters)
     
   - record: platform:hypershift_nodepools:max
-    expr: max by(platform) (hypershift_nodepools)
\ No newline at end of file
+    expr: max by(platform) (hypershift_nodepools)
+
+  - record: hypershift:nodepools:size
```

Comment:
would this record as many NodePools as we have in a platform? e.g in a management cluster with 80 HC and 3 NodePools each that would be 240 time series. 
Do we know what's our cardinality budget for telemetry? cc @csrwng

Reply:
right, other alternative is to use `sum by (platform)` which would yield the total number of nodes across all clusters.nodepools, but I am not sure if that would be useful.